lib/utils/classes.py

```diff
@@ -21,3 +26,157 @@ class Stock:
 
     def __init__(ticker: str):
         self.ticker = ticker
+
+
+
+class myLoginSession:
```

Comment:
```suggestion
class Session:
```
Nit: Naming convention

Reply:
Done